tests/test_indexing.py

```
@@ -811,5 +812,213 @@ def test_stale_path_kept_when_no_live_entry_matches(self, env):
+        suggested, strategy = _suggest_path_correction(
+            "/home/old/myproject", project_data, jsonl_paths,
+        )
+        assert suggested == str(live_dir)
+        assert strategy == "JSONL cwd"
+
+    def test_jsonl_cwd_nonexistent_skipped(self, tmp_path):
+        """JSONL cwd that doesn't exist on disk is skipped."""
+        project_data = {"encodedPaths": ["enc-a"], "name": "myproject"}
+        jsonl_paths = {"enc-a": "/nonexistent/path/myproject"}
+
+        suggested, strategy = _suggest_path_correction(
```

Ruff (0.15.10)

[warning] Unpacked variable `strategy` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)